### PR TITLE
add jar_packager unit test

### PR DESCRIPTION
**Pull request by jsun28**
add an initial version of jar_packager unit test

### CODE REVIEW

File: packaging/tests/test_jar_packager.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import unittest
+import os.path
+from pathlib import Path
+
+from package.jar_packager import create_jar
+from package.connector_file import ConnectorFile
+
+XSD_TEST_FOLDER = Path("tests/test_resources")
```
Comment (pvanderknyff) [inline packaging/tests/test_jar_packager.py]: I think it should be just TEST_FOLDER.

**Comment by pvanderknyff** [comment]
Let's use the same valid connector for both cases. We don't need to have two versions of the same files.

**Comment by jsun28** [comment]
ok cool, will change it

**Comment by jsun28** [comment]
commits are a bit confusing, but file changed should be clear to look at

**Pull request merged**